lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -23,11 +23,11 @@
   }
 
 
-  /*-----------------
+  // -----------------
 
-    Breadcrumbs
+  // Breadcrumbs
 
-  -----------------*/
+  // -----------------
```

Comment:
Why is this also changing comment style?

Not a huge deal, but just to note, this now means the empty lines above are now _outside_ of the comment(s), instead of _inside_. :shrug:

Reply:
The linter doesn't allow comments like this one it seems.

Reply:
:shrug: